source/upgrade.rst

```diff
@@ -65,7 +65,8 @@ Docker
        docker pull qiime2/core:2017.6
 
 Amazon AWS
```

Comment:
Can we update this heading, too? Maybe something like "Amazon AWS (Official QIIME 2 Image)".